src/olympia/addons/tasks.py

```diff
@@ -562,6 +565,21 @@ def add_static_theme_from_lwt(lwt):
     for addon_tag in AddonTag.objects.filter(addon=lwt):
         AddonTag.objects.create(addon=addon, tag=addon_tag.tag)
 
+    # Steal the ratings (even with soft delete they'll be deleted anyway)
+    addon_updates.update(
+        average_rating=lwt.average_rating,
+        bayesian_rating=lwt.bayesian_rating,
+        total_ratings=lwt.total_ratings,
+        text_ratings_count=lwt.text_ratings_count)
+    rating_activity_log_ids = [
+        l.id for l in amo.LOG if getattr(l, 'action_class', '') == 'review']
+    for rating in Rating.unfiltered.filter(addon=lwt):
+        rating.update(addon=addon, version=version)
```

Comment:
`Rating.unfiltered.filter(addon=lwt).update(addon=addon, version=version)` should avoid `post_save` and therefore should avoid recalculating the averages etc that we already know, making the whole thing much faster (less work on celery). Also avoid loading a bunch of `Rating` instances in memory.